<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_window</class>
 <widget class="QDialog" name="settings_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>17</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Saving</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>17</height>
      </size>
     </property>
     <property name="text">
      <string>Number-Formatting</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QRadioButton" name="radioButton_scientific">
     <property name="text">
      <string>Scientific</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QRadioButton" name="radioButton_mixed">
     <property name="text">
      <string>Scientific from 1e...</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_4">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>17</height>
      </size>
     </property>
     <property name="text">
      <string># decimals:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QRadioButton" name="radioButton_plain_numbers">
     <property name="text">
      <string>Plain</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="Line" name="line_2">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="16" column="1" colspan="3">
    <widget class="Line" name="line_4">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QCheckBox" name="checkBox_autosave">
     <property name="text">
      <string>Autosave on closing</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QSpinBox" name="spinBox_scientific_from">
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="17" column="1" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="2">
    <widget class="QSpinBox" name="spinBox_n_decimals">
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label_5">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>17</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Files</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="label_6">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>17</height>
      </size>
     </property>
     <property name="text">
      <string>Python-Files Path</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLabel" name="label_7">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>17</height>
      </size>
     </property>
     <property name="text">
      <string>Measurement-Data Path</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>17</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="3">
    <widget class="Line" name="line_3">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QCheckBox" name="checkBox_dark_mode">
     <property name="text">
      <string>Dark Mode</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2" colspan="2">
    <widget class="Path_Button_Edit" name="pathButton_py_files" native="true"/>
   </item>
   <item row="14" column="2" colspan="2">
    <widget class="Path_Button_Edit" name="pathButton_meas_files" native="true"/>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="label_8">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>17</height>
      </size>
     </property>
     <property name="text">
      <string>Device-Driver Path</string>
     </property>
    </widget>
   </item>
   <item row="15" column="2" colspan="2">
    <widget class="Path_Button_Edit" name="pathButton_device_path" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Path_Button_Edit</class>
   <extends>QWidget</extends>
   <header>utility.path_button_edit</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settings_window</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settings_window</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
